content/en/news/releases/1.6.x/announcing-1.6/announcing-1.6.1/index.md

```diff
@@ -0,0 +1,50 @@
+- **Fixed** `istioctl analyze` to validate `networking.istio.io/v1beta1` resources ([Issue 24064](https://github.com/istio/istio/issues/24064))
+- **Fixed** typo of `istio` in `ControlZ` dashboard log ([Issue 24039](https://github.com/istio/istio/issues/24039))
+- **Fixed** tar name to directory translation ([Issue 23635](https://github.com/istio/istio/issues/23635))
+- **Improved** certificate management for multi-cluster and virtual machine setup from `samples/certs` directory to `install/tools/certs` directory
+- **Improved** `pilot-agent's` handling of client certificates when only a CA client certificate is present
+- **Improved** `istiocl upgrade` to direct users to the `istio.io` website to migrate from `v1alpha1` security policies to `v1beta1` security policies
+- **Fixed** release URL name for `istioctl upgrade`
+- **Fixed** `k8s.overlays` for cluster resources
+- **Fixed** `HTTP/HTTP2` conflict at Gateway ([Issue 24061](https://github.com/istio/istio/issues/24061) and [Issue 19690](https://github.com/istio/istio/issues/19690))
+- **Fixed** Istio Operator to respect the `--operatorNamespace` argument ([Issue 24073](https://github.com/istio/istio/issues/24073))
+- **Fixed** Istio operator hanging when uninstalling Istio ([Issue 24038](https://github.com/istio/istio/issues/24038))
+- **Fixed** TCP metadata exchange for upstream clusters that specify `http2_protocol_options` ([Issue 23907](https://github.com/istio/istio/issues/23907))
+- **Added** `sideEffects` field to `MutatingWebhookConfiguration` for `istio-sidecar-injector` ([Issue 23485](https://github.com/istio/istio/issues/23485))
+- **Improved** installation for replicated control planes ([Issue 23871](https://github.com/istio/istio/issues/23871))
+- **Fixed** `istioctl experimental` pre-check to report compatible versions of Kubernetes (1.14-1.18) ([Issue 24132](https://github.com/istio/istio/issues/24132))
+- **Fixed** Istio operator namespace mismatches causing a resource leak when pruning resources ([Issue 24222](https://github.com/istio/istio/issues/24222))
+- **Fixed** SDS Agent failing to start when Proxy uses file mounted certs for Gateways ([Issue 23646](https://github.com/istio/istio/issues/23646))
+- **Fixed** incorrect `ServiceAccount` being used in custom `IngressGateway` Deployment ([Issue 23303](https://github.com/istio/istio/issues/23303))
+- **Fixed** TCP over HTTP conflicts causing invalid configs ([Issue 24084](https://github.com/istio/istio/issues/24084))
+- **Fixed** the use of external name when remote pilot address is a hostname ([Issue 24155](https://github.com/istio/istio/issues/24155))
+- **Fixed** Istio CNI node `DaemonSet` starting when Istio CNI and `cos_containerd` are enabled on Google Kubernetes Engine (GKE) ([Issue 23643](https://github.com/istio/istio/issues/23643))
+- **Fixed** Istio CNI causing pods to experience a 30-40s delay before starting with the init container ([Issue 23770](https://github.com/istio/istio/issues/23770))
```

Comment:
@stewartbutler Thoughts on phrasing?

Reply:
I'd rephrase as `Istio CNI causing pod intialization to experience a 30-40s delay on startup when DNS unreachable`. It wasn't the init container, it was in the CNI code.

Reply:
Thanks. Changed it.